.github/workflows/create-cli-release.yml

```diff
@@ -18,13 +18,12 @@ on:
 
 jobs:
   check-for-moratorium:
-    if: fromJSON(inputs.isStableCandidate)
-    run: ./scripts/release/tps_check_lock cli ${{ github.sha }}
-    environment: ChangeManagement
-    env:
-      TPS_API_TOKEN: ${{ secrets.TPS_API_TOKEN_PARAM }}
+    if: ${{ fromJSON(inputs.isStableCandidate) }}
+    uses: ./.github/workflows/tps-check-lock.yml
```

Comment:
Should we delete `tps_check_lock` -- https://github.com/heroku/cli/blob/main/scripts/release/tps_check_lock -- now that we switched to `tps-check-lock.yml`?

Reply:
No, because tps-check-lock.yml just runs the tps_check_lock script.

Reply:
I put it in a separate yml file so that we could run it both inside create-cli-release.yml and start-gh-release.yml and so that we can run it manually to test that it is working without having to publish (or try to publish) a release.